scripts/includes/common.sh

```diff
@@ -100,23 +105,26 @@ generate_random_passwords() {
 
 configure_st2_user () {
   # Create an SSH system user (default `stanley` user may be already created)
-  if (! id stanley 2>/dev/null); then
-    sudo useradd stanley
+  SYSTEM_USER=stanley
```

Comment:
I don't think we should assign `stanley` to a var in `curl|bash` installer which is tend to be simple as possible.
I think that `stanley` should be hardcoded everywhere as it was before and keep it simple.

Additionally, keep in mind that `docs` instructions should be modified to be in sync with the installer. So that's another point to have less expressions, constructs and functions.
Think also how you'd modify respective doc section: https://docs.stackstorm.com/install/deb.html#configure-ssh-and-sudo

Not sure what others think? cc @Kami

Reply:
+1